internal/service_maintenance/maintenance.go

```diff
@@ -0,0 +1,48 @@
+package servicemaintenance
+
+import (
+	"context"
+
+	aiven "github.com/aiven/go-client-codegen"
+	aivenservice "github.com/aiven/go-client-codegen/handler/service"
+	"github.com/nais/api/internal/service_maintenance/fake"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	fakeAivenToken = "fake-aiven-token"
+)
+
+type Client interface {
+	ServiceGet(context.Context, string, string, ...[2]string) (*aivenservice.ServiceGetOut, error)
+	ServiceMaintenanceStart(context.Context, string, string) error
+}
+
+type Manager struct {
+	client Client
+	log    *logrus.Entry
+}
+
+func NewManager(ctx context.Context, token string, log *logrus.Entry) (*Manager, error) {
+	if token == fakeAivenToken {
```

Comment:
Kan vi heller konfigurere dette ved hjelp av `WITH_FAKE`? https://github.com/nais/api/blob/main/internal/cmd/api/config.go#L181

Reply:
Burde fått ryddet opp i de servicene som ikke har flyttet til det nye patterne. For vi baserte vår del på Vulnerabilities som ikke gjorde det.